functionary/ui/templates/builder/build_list.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% block content %}
+    <div>
+        {% for build in object_list %}
+            <div class="block">
+                <div class="card">
+                    <div class="card-content">
+                        <div class="media">
+                            <div class="media-left">
+                                <span class="icon is-large">
+                                    <i class="fa fa-2x fa-wrench"></i>
+                                </span>
+                            </div>
+                            <div class="media-content">
+                                <a href="{% url 'ui:build-detail' build.id %}">
+                                    <p class="title is-4">{{ build.id }}</p>
```

Comment:
In places where you're currently displaying the `build.id`, I think it's probably more useful to show the `build.package.name`.  That's the thing that is going to be recognizable to people.  I think that's here and on the build detail page as well.

Also, I think there's other information we could display in the list view that would be useful:

* created_at
* creator
* status

Take a look at how the task list view is formatted.  Something similar would probably work here.